playbooks/roles/configure/tasks/configure_instance.yml

```diff
@@ -60,5 +60,11 @@
     echo "ZWED_agent_jwt_fallback=true" >> "{{ zowe_instance_dir }}/instance.env"
     echo "ZWED_agent_jwt_enabled=true" >> "{{ zowe_instance_dir }}/instance.env"
 
+# If zowe_apiml_caching_service_start true,
+# caching service is deployed on marist along with the rest of API ML and its sanity tests are run
+- name: Add Caching Service start parameter to instance.env
+  raw: |-
+    echo "zowe_apiml_caching_service_start=true" >> "{{ zowe_instance_dir }}/instance.env"
```

Comment:
You are injecting `zowe_apiml_caching_service_start` here, but the envar the the apiml use to know whether to start is `ZOWE_CACHING_SERVICE_START` so this won't work

Reply:
Yes you're right, I will fix it